modules/core/src/main/java/org/apache/ignite/internal/direct/stream/v2/DirectByteBufferStreamImplV2.java

```diff
@@ -301,6 +301,9 @@
     /** */
     protected boolean lastFinished;
 
+    /** byte-array representation of string */
+    private byte[] byteArr;
```

Comment:
Can we perhaps make it a `private volatile` and name smth like `curStrBackingArr`? I'm not confident that it will be called from the same thread.

Reply:
@akalash says volatile is not needed. Can you please just rename this field as you see fit?